lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -149,3 +157,58 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
     visibility: hidden;
   }
 }
+
+.nx-global-sidebar__other-content {
+  @include container-vertical;
+
+  flex-grow: 1;
+  padding: $nx-spacing-md $nx-global-sidebar-left-right-spacing;
+  max-height: unset;
+
+  + .nx-global-sidebar__footer {
+    border-top: 1px solid $nx-global-sidebar-separator-color;
+    margin-top: 0;
+  }
+}
+
+.nx-global-sidebar__footer {
+  @include container-vertical;
+
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: auto;
+  padding: $nx-spacing-xxl $nx-global-sidebar-left-right-spacing 0 $nx-global-sidebar-left-right-spacing;
+  text-align: center;
+
+  &:empty {
+    padding-top: 0;
```

Comment:
We'll need a fully empty example to run applitools on.  Maybe don't link to it from the documentation page though since it isn't an endorsed way of using the component

Reply:
> We'll need a fully empty example to run applitools on. Maybe don't link to it from the documentation page though since it isn't an endorsed way of using the component

https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/8029b2495340ea0811ab01a00a0ebe452bd1d92a

I also captured the minimal example, and left a note that the full examples (plus open/closed states) are captured by the `NxGlobalSidebar`